com.unity.render-pipelines.high-definition/Samples~/LocalVolumetricFogMasks/.sample.json

```diff
@@ -0,0 +1,5 @@
+{
+    "displayName":"Local Volumetric Fog Mask Textures",
+    "description": "Adds a set of 3D mask textures to your project. These texture can be used for the Density Mask Texture input of the Local Volumetric Fog.",
```

Comment:
Suggested change: 

Adds a set of 3D Mask Textures to your project. You can assign these Textures to the Local Volumetric Fog's Density Mask Texture input.